2022/_data/jobs.yml

```diff
@@ -445,5 +445,38 @@
         "link": "https://www.youtube.com/watch?v=-4digbYIFxs&list=PLKrSWEllR0Lg5c0eYT5AEi0vamlAZHilb"
       }
     ]
+  },
+  {
+    "company": {
+      "name": "Ocado Technology",
+      "logo-url": "/assets/img/sponsors/ocadotechnology-200px-h.png",
+      "description": "At Ocado Technology, we are pioneers focusing on solving some of the toughest tech challenges of our age. Heavily invested in Java, we use this language in our end-to-end integrated platform Ocado Smart Platform (OSP), enabling the world’s largest retailers to offer the best customer experiences.",
+      "level": "premium"
+    },
+    "links": {
+      "web": "https://ocadotechnology.co/barcelona",
+      "twitter": "https://twitter.com/OcadoTechnology",
+      "linkedIn": "https://www.linkedin.com/company/ocado-technology"
+    },
+    "vacancies": [
+      {
+        "title": "Java Software Engineer",
+        "location": "Barcelona (hybrid)",
+        "description": "Would you like to build real products, beyond just writing code? Your Java skills make a difference to millions of users globally. Join us to change the way the world shops.<br/><br/><strong>What would you be doing?</strong><br/>- Working with the following Tech stack: Java, Python, Kotlin, Swift, Firebase, Terraform, Docker, AWS, Google Cloud Platform, ELK, New Relic, Cloudwatch, Prometheus, Grafana and more<br/>- Developing new systems and capabilities as well as improving and developing existing systems <br/>- Working with the team on continuous improvement of the Software Development Life Cycle<br/>- Playing a key part in the technical design of solutions to interesting problems<br/>- Promoting good engineering practices, bug prevention strategies, testability, fixing security vulnerabilities where appropriate, and other advanced quality concepts<br/><br/><strong>You're a great fit if you have:</strong><br/>- Demonstrable industry programming experience with knowledge of Java<br/>- Passion for developing high-quality enterprise software<br/>- Eagerness to take an active role in supporting the business needs<br/>- OOP and software design skills<br/>- Passion and understanding of TDD and Clean Code<br/>- Experience working in a continuous delivery environment<br/>- Knowledge of AWS/Cloud environments<br/>- A BSc degree in Computer Science (or a related field) or equivalent industry experience<br/>- Good command of English (spoken and written)<br/>",
```

Comment:
`BSC` all in capital

Reply:
Actually, it is BSc or MSc. 
However, that's the text entered by the sponsor. I did not want to change their words. I thought they had to review their texts themselves.

Reply:
You always learn something new. Thanks!

Reply:
> I thought they had to review their texts themselves.

Same as anyone, as you mention in case of doubt, let's leave as it is, but clear mistakes I think we can correct.